src/client/mod.rs

```diff
@@ -390,7 +389,7 @@ impl KafkaClient {
     pub fn new(hosts: Vec<String>) -> KafkaClient {
         KafkaClient {
             config: ClientConfig {
-                client_id: CLIENTID.to_owned(),
+                client_id: String::new(),
```

Comment:
yes, i was thinking about keeping "kafka-rust" as the default in the first place - this is the client id which has been sent so far. however, the id is in many cases not really useful and it's few bytes added to every request. therefore i though, it be better left to the user to decide whether she does want a client id or not.

interestingly, the java based `KafkaConsumer` and `KafkaProducer` ensure to always generate a client-id. though i believe this has to do with metrics gathering on the client side when running multiple consumers, producers within one JVM.